src/sentry/integrations/slack/tasks.py

```diff
@@ -170,6 +175,10 @@ def find_channel_id_for_alert_rule(organization_id, uuid, data, alert_rule_id=No
             redis_rule_status.set_value("failed")
             return
 
+    # This function call will write "input_channel_id" to the appropriate actions before we create the serializer
+    # I wasn't sure of a way to use the same loop to return True in the endpoints and to also do the lookup and didn't want to duplicate the loop
```

Comment:
@iProgramStuff I wouldn't worry about duplicating the loop, better to have a bit of repetition if we can have cleaner fucntions.